r2-streamer-swift/Parser/EPUB/EPUBParser.swift

```diff
@@ -111,10 +111,26 @@ final public class EpubParser: PublicationParser {
         if ((try? container.data(relativePath: EPUBConstant.lcplFilePath)) != nil) {
             return DRM(brand: .lcp)
         }
-        
-        return DRM(brand: .adobe)
-        
-        // return nil
+        return nil
+    }
+  
+    
+    /// Check license files at the book location.
+    /// Adobe or LCP DRM licence can be a separate file, not included into .epub archive.
+    /// - Parameter url: File URL
+    /// - Returns: A DRM, if any found.
```

Comment:
can you mention here that this only returns adobe or lcp DRM, and nil otherwise?

Reply:
sure, added